templates/hpe/index.html

```diff
@@ -171,16 +175,18 @@ <h2>Proven results from collaborative solutions</h2>
   </div>
 </section>
 
-<section class="p-section u-image-position">
-  <div class="u-fixed-width">
-    <hr class="p-rule">
-  </div>
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
   <div class="row">
-    <div class="col-6 col-medium-3">
-      <h2 id="events">Events</h2>
-      <p>Canonical participates in international events with HPE to show the partnership's value and the offerings we provide, from private cloud to edge, to AI/ML, to open source applications security.</p>
+    <div class="col-6 col-medium-2">
+      <div class="p-block">
```

Comment:
Wait, are we wrapping every piece of text now in a p-block div? @bartaz is this expected usage?

Reply:
No. Block is supposed to be used only if a section (p-strip or p-section) is further divided into logical blocks of content.

Looking at the design I see possibly 2 use cases for blocks:

Here the "HPE Discover 2023" and "HPE OBSS APAC 2023" blocks with their descriptions are likely 2 block components (as they are subsections of their "Recent events".

<img width="436" alt="image" src="https://github.com/canonical/ubuntu.com/assets/83575/1b0dabe7-fc52-4580-b237-25c57983a5fb">

And similarly blogs seem to be 2 blocks:
<img width="435" alt="image" src="https://github.com/canonical/ubuntu.com/assets/83575/064d7b84-f645-4742-a489-13a769923e04">

Block component should not be used to wrap around single headings or paragraphs of text.

Reply:
We add it where we need to insert space to prevent adjacent elements from distracting the viewer while reading this text. In this case, This is what happens without it:

![image](https://github.com/canonical/ubuntu.com/assets/2741678/4f9fc17b-3c33-4742-8b62-8ac3c5e2f4c8)

And this is what it looks like with it: 

![image](https://github.com/canonical/ubuntu.com/assets/2741678/8925bed8-8dd3-42c9-8366-0b48dcdae6e3)

Reply:
Very good points @bartaz, I've refactored it so these are now p-blocks, as per the documentation:
![image](https://github.com/canonical/ubuntu.com/assets/2741678/cc5d01a0-e2b1-4e8e-996e-923726a14fb2)

Same visual effect but structured to make more sense.

Reply:
We are also using `p-blocks` for titles on the left